gstcefsrc.cc

```diff
@@ -397,6 +397,17 @@ run_cef (GstCefSrc *src)
   GST_INFO  ("Initializing CEF");
 
   gchar* base_path = get_plugin_base_path();
+
+  // If not absolute path append to current_dir
+  if (!g_path_is_absolute(base_path)) {
+    gchar* current_dir = g_get_current_dir();
```

Comment:
you're missing a free for that one :)

Reply:
Good catch. How do people even make C programs without leaking memory :D

Reply:
valgrind + tests + years of suffering would be the answer